Tests/BusinessLogic.Tests/UnitTests/LogicTests/PlayedGamesTests/PlayedGameRetrieverTests/SearchPlayedGamesTests.cs

```diff
@@ -146,7 +148,7 @@ public void ItSetsAllTheFields()
             };
             autoMocker.Get<IDataContext>().Expect(mock => mock.GetQueryable<PlayedGame>()).Return(playedGames.AsQueryable());
 
-            var results = autoMocker.ClassUnderTest.SearchPlayedGames(new PlayedGameFilter()).First();
+            var results = autoMocker.ClassUnderTest.SearchPlayedGames(new PlayedGameFilter(), null).First();
```

Comment:
Not a huge fan of using `null` here but I didn't have an existing `ApplicationUser`. I can add one if you'd prefer